src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -436,6 +489,8 @@ def _process_genome_attri_tools(genome_attr_tools: set[str],
         docs.extend(parse_ops(root_dir, kbase_collection, load_ver))
 
     docs = merge_docs(docs, '_key')
+    if fatal_ids:
+        docs = _remove_docs_with_fatal_ids(docs, fatal_ids)
```

Comment:
I was thinking we'd avoid processing the record at all, but if this works for now I guess that's ok. The risk is in the future the code may choke on missing data in the `parse_ops` method

Reply:
I agree now if we need to check all tools.

Reply:
Updated!

Reply:
:+1: